### ISSUE TITLE
[scheduler] Want to use MultiStepLR scheduler

### ISSUE THREAD

**Issue by raymin0223** [Lee-Gihun/Pytorch-OSI-setup#19]
While doing some experiments, I sometimes want to use `MultiStepLR` scheduler.

If you write **list type** value on `opt.scheduler.step`, model will use that list on **milestones** argument of `MultiStepLR`.

It is related to commit e04786cd6f4eda1ba2e0c4ebac9e04af277ba70a  :)

**Comment by raymin0223** [comment]
It is related to commit e04786cd6f4eda1ba2e0c4ebac9e04af277ba70a.

**Comment by raymin0223** [comment]
I will close this issue.
